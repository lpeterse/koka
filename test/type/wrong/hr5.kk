val ids = [id : forall<a> a -> a]

val ids2 = (ids : forall<a> list<forall<b> b -> a>)