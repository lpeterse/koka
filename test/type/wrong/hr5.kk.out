test/type/wrong/hr5.kk(3,13): error: types do not match
 context : ids : forall<a> list<forall<b> b -> a>
 term : ids
 inferred type: list<forall<a> (a) -> a>
 expected type: forall<a> list<forall<b> (b) -> a>
 because : type cannot be instantiated to match the annotation